netjsonconfig/backends/openwrt/converters/interfaces.py

```diff
@@ -117,21 +120,24 @@ def __intermediate_addresses(self, interface):
             static[address_key].append('{address}/{mask}'.format(**address))
             static.update(self.__intermediate_address(address))
         if static:
-            # do not use CIDR notation when using a single ipv4
-            # see https://github.com/openwisp/netjsonconfig/issues/54
-            if len(static.get('ipaddr', [])) == 1:
-                network = ip_interface(static['ipaddr'][0])
-                static['ipaddr'] = str(network.ip)
-                static['netmask'] = str(network.netmask)
-            # do not use lists when using a single ipv6 address
-            # (avoids to change output of existing configuration)
-            if len(static.get('ip6addr', [])) == 1:
-                static['ip6addr'] = static['ip6addr'][0]
-            result.append(static)
+            result.append(self.__intermediate_static_address(static))
```

Comment:
I had to move the logic above to this method because flake8 was complaining about cyclomatic complexity due to the 2 lines I had to add above.